templates/base/footer.tmpl

```diff
@@ -66,5 +66,6 @@
 	<!-- JavaScript -->
 	<script src="{{AppSubUrl}}/vendor/plugins/semantic/semantic.min.js"></script>
 	<script src="{{AppSubUrl}}/js/index.js?v={{MD5 AppVer}}"></script>
+{{template "base/footer_extras" .}}
```

Comment:
Cleaner to create a folder extra at root of templates containing footer.tmpl and header.tmpl.